03-transactions.md

```diff
@@ -171,9 +174,25 @@ After 16 blocks, anyone can sweep the anchor with witness:
 
     <>
 
+#### `shared_anchor` Output (zero_fee_commitments)
+
+This output can be spent by anyone to provide incentive to mine the transaction, using child-pays-for-parent (since the commitment transaction doesn't pay any mining fees).
+This output is only added when the commitment transaction is using version 3 (`zero_fee_commitments`) and prevents pinning attacks on the commitment transaction.
+It is using the following standard P2A (pay-to-anchor) script:
+
+    OP_1 <0x4e73>
+
+The amount of this output is usually 0 sats, in which case it must be spent by the child transaction paying the mining fees.
+When the commitment contains [trimmed outputs](#trimmed-outputs), their amount is added to this `shared_anchor` output until it reaches 240 sats.
```

Comment:
nit: I think it would be helpful to make a note here explaining why need to switch between the anchor and fees
* Ephemeral dust requires zero fees for values <= dust limit to be standard
* The dust limit for P2A is 240

Perhaps link to this issue: https://github.com/bitcoin/bitcoin/issues/31938 (helped me understand it better)